catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-left/navigation-left.view.js

```diff
@@ -55,5 +57,15 @@ module.exports = Marionette.LayoutView.extend({
             return !source.get('available');
         });
         this.$el.toggleClass('has-unavailable', hasDown);
+    },
+    handleLogo: function() {
+        var hasLogo = properties.showLogo && properties.ui.vendorImage !== "";
+        this.$el.toggleClass('has-logo', hasLogo);
+    },
+    serializeData: function(){
+        return {
+            logo: properties.ui.vendorImage,
+            showLogo: properties.showLogo && properties.ui.vendorImage !== ""
```

Comment:
The logic in the line is identical to Line 62. Can you refactor it a new method?

Reply:
done